apps/web/src/data/ecosystem.json

```diff
@@ -1363,7 +1363,7 @@
     "name": "Synapse Protocol",
     "url": "https://www.synapseprotocol.com",
     "description": "Synapse is comprised of a cross-chain messaging framework and an economically secure method to reach consensus on the validity of cross-chain transactions, enabling developers to build truly native cross-chain apps.",
-    "imageUrl": "/images/partners/synpase.webp",
+    "imageUrl": "/images/partners/synapse.webp",
```

Comment:
The image file would also need to be updated to ensure there isn't a broken image